Resources/Prototypes/Catalog/Fills/Lockers/medical.yml

```diff
@@ -79,33 +78,32 @@
   components:
   - type: StorageFill
     contents:
-      - id: HandheldHealthAnalyzer
-        prob: 0.6
-      - id: ClothingHandsGlovesLatex
-      - id: ClothingHeadsetMedical
-     # - id: ClothingEyesHudMedical #Removed until working properly
-      - id: ClothingBeltMedical
-      - id: ClothingHeadHatSurgcapGreen
-        prob: 0.1
-        orGroup: Surgcaps
-      - id: ClothingHeadHatSurgcapPurple
-        prob: 0.05
-        orGroup: Surgcaps
-      - id: ClothingHeadHatSurgcapBlue
-        prob: 0.90
-        orGroup: Surgcaps
-      - id: UniformScrubsColorBlue
-        prob: 0.5
-        orGroup: Surgshrubs
-      - id: UniformScrubsColorGreen
-        prob: 0.1
-        orGroup: Surgshrubs
-      - id: UniformScrubsColorPurple
-        prob: 0.05
-        orGroup: Surgshrubs
-      - id: ClothingMaskSterile
-      - id: HandLabeler # Frontier
-      - id: ClothingBeltMedical # Frontier
+#      - id: HandheldHealthAnalyzer
+#        prob: 0.6
+#      - id: ClothingHandsGlovesLatex
+#      - id: ClothingHeadsetMedical # Frontier
+      - id: ClothingEyesHudMedical # Frontier
+#      - id: ClothingBeltMedical
+#      - id: ClothingHeadHatSurgcapGreen
+#        prob: 0.1
+#        orGroup: Surgcaps
+#      - id: ClothingHeadHatSurgcapPurple
+#        prob: 0.05
+#        orGroup: Surgcaps
+#      - id: ClothingHeadHatSurgcapBlue
+#        prob: 0.90
+#        orGroup: Surgcaps
+#      - id: UniformScrubsColorBlue
+#        prob: 0.5
+#        orGroup: Surgshrubs
+#      - id: UniformScrubsColorGreen
+#        prob: 0.1
+#        orGroup: Surgshrubs
+#      - id: UniformScrubsColorPurple
+#        prob: 0.05
+#        orGroup: Surgshrubs
+#      - id: ClothingMaskSterile
+      - id: ClothingBeltParamedicFilled # Frontier
```

Comment:
again, this isnt a paramedic locker, please dont do this. And leave the meds and gloves and scanner in please. I dont want a hundred lockers with only a paramedic belt in them.

Reply:
Used wrong belt by mistake.
The scanners are in nano meds, gloves in drobs,
Ill add them anyway for faster access.

Reply:
gloves are fine to stay removed (for now, until we bring back diseases) but put the scanner back in imo.  Same idea, if a monkey needs to do the job they need the fundamental/essential tools, just not the fluff

Reply:
Fixed

Reply:
its still a paramedic belt